lnwire/node_announcement.go

```diff
@@ -164,34 +164,19 @@ func (c *NodeAnnouncement) Command() uint32 {
 //
 // This is part of the lnwire.Message interface.
 func (c *NodeAnnouncement) MaxPayloadLength(pver uint32) uint32 {
-	var length uint32
-
 	// Signature - 64 bytes
-	length += 64
-
 	// Timestamp - 4 bytes
-	length += 4
-
-	// Ipv6 - 16 bytes
-	length += 16
-
-	// Port - 4 bytes
-	length += 4
-
+	// NumAddresses - 2 bytes
+	// AddressDescriptor - 1 byte
+	// Ipv4 - 4 bytes
+	// Port - 2 bytes
 	// NodeID - 33 bytes
-	length += 33
-
 	// RGBColor - 3 bytes
-	length += 3
-
 	// pad - 2 bytes
-	length += 2
-
 	// Alias - 32 bytes
-	length += 32
 
-	// 158
-	return length
+	// Base size, 147, but can be variable due to multiple addresses
```

Comment:
Maybe it is not so important as far as it will change in near future , but it seems for me that base size should be `140` because length of addresses might be equal to `0` according to spec.